src/pudl/transform/eia923.py

```diff
@@ -1223,3 +1224,59 @@ def _core_eia923__fuel_receipts_costs(
     frc_df = frc_df.dropna(subset=["energy_source_code"])
 
     return frc_df
+
+
+@asset
+def _core_eia923__cooling_system_information(
+    raw_eia923__cooling_system_information: pd.DataFrame,
+) -> pd.DataFrame:
+    """Transforms the eia923__cooling_system_information dataframe.
+
+    Applies typical NA conversion and date conversion.
+
+    As of 2024-02-28: 2008 and 2009 only have annual rates, but the
```

Comment:
Rather than dropping this data, I'd make an annual rate column for each of these and just move the 2008-9 data in there. There's no good reason to drop the values, even though they shouldn't be in the same column as the rest of the data for legibility.

Reply:
There's already an "annual" columns that only have non-null values for both of these years, but it's the raw cfs value instead of GPM - do you mean we should make an annual GPM column and remove the annual CFS column?

Reply:
So the raw data reports values in both columns for 2008-9 data? Are they different than one another?
I think the ideal outcome would be one monthly and one annual column with the same units. Whether we fill the annual column with monthly data I feel less set on.

Reply:
Raw data only reports annual data for 2008-09; and then it only reports non-annual data for 2010+